src/sdk/pynni/tests/assets/search_space.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
It's nice job! In the same time, can we add some code for validate the json file of search space?

Reply:
Precondition check will not be included in this PR